content/docs/buildpack-author-guide/create-buildpack/make-buildpack-configurable.md

```diff
@@ -128,3 +128,5 @@ Now that you've finished your buildpack, how about extending it? Try:
 - Caching the downloaded Ruby version
```

Comment:
Given that we have a follow up step I don't think this section is applicable anymore or maybe we can add packaging as a link in going further.

Reply:
Removed